general/development/policies.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Policies
+tags:
+  - Policies
+  - Coding guidelines
+  - Developer processes
+---
+
+:::note
+
+This page is the top-level page describing Moodle's coding guidelines.  It's the place to start if you want to know how to write code for Moodle.
+
+:::
+
+## Moodle architecture
+
+Moodle tries to run on the widest possible range of platforms, for the widest possible number of people, while remaining easy to install, use, upgrade and integrate with other systems.
+
+:::info
+
+For more about this, see [Moodle architecture](https://docs.moodle.org/dev/Moodle_architecture).
+
+:::
+
+## Plugins
+
+Moodle has a general philosophy of modularity.  There are nearly 30 different standard types of plugins and even more sub-plugin types, however all of these plugin types work the same way. Blocks and activities are the only small exceptions.
+
+:::info
+
+For more about this, see [Moodle plugins](https://docs.moodle.org/dev/Plugins) and [Moodle sub-plugins](https://docs.moodle.org/dev/Subplugins).
+
+:::
+
+## Coding style
+
+Consistent [coding style](/general/development/policies/codingstyle) is important in any development project, and particularly so when many developers are involved. A standard style helps to ensure that the code is easier to read and understand, which helps overall quality.
+
+Writing your code in this way is an important step to having your code accepted by the Moodle community.
+
+:::info
+
+For more about this, see the [Moodle coding style](/general/development/policies/codingstyle).
+
+:::
+
+## Security
+
+Security is about protecting the interests and data of all our users.  Moodle may not be banking software, but it is still protecting a lot of sensitive and important data such as private discussions and grades from outside eyes (or student hackers!) as well as protecting our users from spammers and other internet predators.
+
+It's also a script running on people's servers, so Moodle needs to be a responsible Internet citizen and not introduce vulnerabilities that could allow crackers to gain unlawful access to the server it runs on.
+
+:::info
+
+Any single script (in Moodle core or a third party module) can introduce a vulnerability to thousands of sites, so it's important that all developers strictly follow our [Moodle security guidelines](/general/development/policies/security).
+
+:::
+
+## HTML5 and CSS
```

Comment:
Consider renaming this since it's not just HTML5 and CSS, but also WCAG, ARIA, etc.

```suggestion
## Standards
```